app/controllers/DeclarationController.scala

```diff
@@ -45,7 +46,15 @@ class DeclarationController @Inject()(
   private val logger = Logger(this.getClass)
 
   def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
-    Future.successful(Ok(view()))
+
+    val allSectionsCompleted =
```

Comment:
using this piece of code in more than one place so plz move this code to LocalBaseController